v1/api/swagger/components/schemas/FCParameters.yaml

```diff
@@ -0,0 +1,14 @@
+type: object
+required:
+  - HostIPAddress
+  - InitiatorName
+properties:
+  HostIPAddress:
+    type: string
+    description: The IP address of the host in dotted notation.
+    example: 10.1.2.3
+  InitiatorName:
+    type: string
+    description: >-
+      FC Initiator world wide port names.
```

Comment:
This is confusing. You either specify this field as a FC initiator or HBA wwpn.

Reply:
Ok, I will keep FC initiator name.

Reply:
I think about it. The current backend can provide in term of attachment is the host ID and volume ID, nothing more. That's something that you can only mapping over to project API. The description of this field needs to revised to reflect the nature to volume attachment.